Dockerfile.openpilot_base

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:20.04@sha256:ed4a42283d9943135ed87d4ee34e542f7f5ad9ecf2f244870e23122f703f91c2
```

Comment:
what about pulling the latest? we don't want to have to manually update this

Reply:
ubuntu:20.04 *should* pull the latest version, but github has locally cached versions that seem to override this. I'll see if I can find a better way to do this

Reply:
can't we `docker pull` it?

Reply:
ah, yea I think we just needed to add --pull to buildx build